app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/subscription/check_your_details.scala.html

```diff
@@ -40,8 +40,8 @@
 )
 
 @(details: SubscriptionDetails)(implicit request:Request[_], messages:Messages, appConfig: ViewConfig)
-
-@title = @{messages("subscription.title")}
+@affinity = @{details.name.fold(a => "organisation", b => "individual")}
```

Comment:
this isn't broken or anything, but you could just put `_` instead of `a` and `b` if you don't care about them

Reply:
done